src/_guides/bump-sh-tutorials/mock-server-with-beeceptor.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Mocking APIs with Beeceptor - Fast, Context-Aware
```

Comment:
I think it’s important to use a more neutral title (the “fast, context-aware” part doesn’t seem necessary to me).